libs/design-system/ui-section/src/lib/ui-section.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="ui-section-card">
+  <img alt="app-image" [src]="imagePath()" class="app-image" />
+  <div class="ui-content-main">
+    <div class="ui-content">
+      <hra-product-logo [id]="logo()" class="product-logo" />
+      <div class="ui-content-header">
+        <h1>
+          {{ tagline() }}
+        </h1>
+        @if (appStatus()) {
```

Comment:
Update to `(appStatus(); as status)` then pass `status` to the component instead of using non-null (`!`) assertions